chapter_local/nyc.md

```diff
@@ -50,6 +37,16 @@ permalink: /nyc/
 
 <hr />
 
+<div class="flex justify-between ai-center pad">
```

Comment:
`ai-center` is such an intriguing name!

Reply:
atomic class names are just descriptive. sometimes i see this as items-center, but that's all it is. it **a**ligns **i**tems along the vertical center

Reply:
i'd rather have tailwind just auto-generate all these and never think about css again, personally. for a future version maybe